playbook/app/pb_kits/playbook/pb_currency/_currency.jsx

```diff
@@ -69,6 +71,24 @@ const Currency = (props: CurrencyProps) => {
     className
   )
 
+  const getFormattedNumber = (input) => new Intl.NumberFormat('en-US', {
+    notation: 'compact',
+    maximumFractionDigits: 1,
+  }).format(input)
+
+  type AbbrType = 'amount' | 'unit'
+
+  const getAbbreviatedValue = (abbrType: AbbrType) => {
+    const num = `${getFormattedNumber(whole.split(',').join(''))}`,
+      isAmount = abbrType === 'amount',
+      isUnit =  abbrType === 'unit'
+    return isAmount ? num.slice(0, -1) : isUnit ? num.slice(-1) : ''
+  }
+
+  const getAmount = abbreviate ? getAbbreviatedValue('amount') : whole,
+    getAbbreviation = abbreviate ? getAbbreviatedValue('unit') : null,
+    getDecimalValue = abbreviate ? '' : `.${decimal}`
```

Comment:
I'm curious why you don't  define a variable with a `const` keyword for last two examples?

Reply:
I actually saw this done a couple times in other files. When you have a bunch of `const`s one after the other, you can separate them by commas and old put one `const` before the first one @JelenaD1

Reply:
*only

Reply:
I did not know that. Thanks for explaining.

Reply:
Mostly just to keep word trash down in the code. Seeing `const` a bunch of times (unnecessarily) is going to reduce readability and make the code appear more monotonous. What is best about this approach is you can now visually group variables of concern together.